docs/support-bundle-guide.md

```diff
@@ -0,0 +1,247 @@
+apiVersion: crd.antrea.io/v1alpha1
+kind: SupportBundleCollection
+metadata:
+  name: collect-support-bundle-for-vm-external-node
+spec:
+  externalNodes:
+    nodeNames: # Empty list means all the ExternalNodes in the Namespace
+      - vm
+    nodeSelector:
+      matchLabels:
+        app: db   # Collect support bundle files from the ExternalNodes which have been labeled as "app: db" in the Namespace
+    namespace: vm-ns # namespace is mandatory when collecting files from ExternalNodes.
+  fileServer:
+    url: yourtestdomain.com:22/root/test # Scheme sftp can be omitted
+  authentication:
+    authType: "BasicAuthentication"
+    authSecret:
+      name: support-bundle-secret
+      namespace: default # antrea-controller has been given the permission to read secrets in "default" namespace.
+EOF
+```
+
+More details about the supported fields in a "SupportBundleCollection" CR, please
+refer to the [CRD definition](/build/charts/antrea/crds/supportbundlecollection.yaml)
+
+You can check the status of `SupportBundleCollection` by running
+`kubectl get supportbundlecollections $Name-of-SupportBundleCollection -ojson`. The following example
+shows a successful creation of a `SupportBundleCollection`. `desiredNodes` shows the number of
+Nodes/ExternalNodes is expected in this request, while `collectedNodes` shows the number of
+Nodes/ExternalNodes which has already uploaded bundle files to the target file server.`collectedNodes`
+and `desiredNodes` should match the number of Nodes/ExternalNodes you want to collect support
+bundle if collection is running successfully.
+
+If following two conditions are presented, it means a bundle creation is succeeded,
+
+1. "Completed" is true
+2. "CollectionFailure" is false.
+
+If any expected Node/ExternalNode is failed to upload the bundle files in the required time, the
+"CollectionFailure" condition is marked as true.
+
+```bash
+$ kubectl get supportbundlecollections collect-support-bundle-for-kube-nodes -ojson
+
+...
+ "status": {
+        "collectedNodes": 1,
+        "conditions": [
+            {
+                "lastTransitionTime": "2022-12-08T06:49:35Z",
+                "status": "True",
+                "type": "Started"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:41Z",
+                "status": "True",
+                "type": "BundleCollected"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:35Z",
+                "status": "False",
+                "type": "CollectionFailure"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:41Z",
+                "status": "True",
+                "type": "Completed"
+            }
+        ],
+        "desiredNodes": 1
+    }
+```
+
+The output contains three tarballs. To access these files, please SSH into the ftp server
+`yourtestdomain.com` with port 22 or scp them to any other storages you used in CR. There will be
```

Comment:
Why you need to SSH to a SFTP server, rather than use "SFTP"? Maybe just say "you can download the files from the SFTP server"